### ISSUE TITLE
mwaa 2.2 does not work on Macs on M1 M2 arm chips

### ISSUE THREAD

**Issue by shmcsensei** [aws/aws-mwaa-local-runner#272]
This line causes the following issue on arm based Mac books.

https://github.com/aws/aws-mwaa-local-runner/blob/a6190befc60eeda41dc6793c731dd648d0bcfae2/docker/docker-compose-local.yml#LL18C9-L18C32

```
Image with reference amazon/mwaa-local:2_2 was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64/v8
```

**Comment by Mercury2699** [comment]
This line was removed in 2.4.3+, making it compatible with other architecture. 
Feel free to remove it when you use local runner for an older version.

**Comment by shmcsensei** [comment]
Isn't the point of v2.2 to match with Airflow version 2.2 in the AWS environment? It may not be because branch v2.2.2 loads airflow v2.6.x .. Its really confusing. 

Most AWS Infrastructure is version fixed, RDS PG v10.1.1 will deploy PGv10.1.1 . How is it that version 2.2.2 of managed airflow deploys v2.6.x ?

My point is that asking me to use version  2.4.3+ is not the solution if I need v2.2 compatibility at an enterprise. I also expect a little better since this is a paid service.

**Comment by Mercury2699** [comment]
> Isn't the point of v2.2 to match with Airflow version 2.2 in the AWS environment?


Yes, therefore, if you want to local runner to match with AWS environment as much as possible, you should just use x86 platform instead. 

> How is it that version 2.2.2 of managed airflow deploys v2.6.x ?


You might have misinterpreted what I said. I meant you can still use v2.2.2 branch, and just simply remove that `platform: "linux/amd64"` line if you still want to use it on Apple Silicon devices. I am not asking you to use v2.4.3 branch, although using a newer version is indeed recommended.

**Comment by sukoneck** [comment]
this worked for me. would a PR be welcome or are do you want it left in?

**Comment by shmcsensei** [comment]
Would changing it to `linux/arm64/v8` be better?
